birdhouse/optional-components/stac-public-access/docker-compose-extra.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.4"
```

Comment:
This works... but if we want to follow the pattern we use elsewhere we should really add this to `birdhouse/optional-components/stac-public-access/config/magpie/docker-compose-extra.yml` so that it will only be set if magpie is enabled as well.

We're almost certainly going to make magpie a required component but it would be nicer to keep the pattern we've already established. 

Thanks for finding that @tlvu